docs/graphql/core/actions/clean-up-actions-logs.rst

```diff
@@ -0,0 +1,47 @@
+.. meta::
+   :description: clean up actions logs
+   :keywords: hasura, docs, actions, clean up, async actions
+
+.. _clean_up_actions_logs:
+
+Clean up actions logs
+=====================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+Hasura stores actions logs from asynchronous operations. If there are a lot of logs, the tables can get huge and you may want to prune them.
+You can use any of the following options to prune your logs depending on your need.
+
+The table involved
+------------------
+
+Actions has a table managed by Hasura:
+
+- ``hdb_catalog.hdb_action_log``: This is the table that stores all captured logs.
+
+Option 1: Get all logs
+----------------------
+.. code-block:: SQL
+   
+   SELECT * FROM hdb_catalog.hdb_action_log;
```

Comment:
For these statements we need some guidance on how to run these. We should introduce tabs also to show console / cli / api workflow? You can check the other actions pages for guidance.